packages/react-meteor-data/index.js

```diff
@@ -8,7 +8,9 @@ if (Meteor.isDevelopment) {
   }
 }
 
-export { default as useTracker } from './useTracker';
-export { default as withTracker } from './withTracker.tsx';
+import useTracker from './useTracker';
+import withTracker from './withTracker.tsx';
 export { useFind } from './useFind';
 export { useSubscribe } from './useSubscribe';
+
+export { useTracker, withTracker };
```

Comment:
1. I'd add a comment why it's needed, as a future ESLint config may suggest to revert it.
2. Why it's not a problem for `useFind` and `useSubscribe`? Maybe changing `useTracker` and `withTracker` to use named exports would help?

Reply:
1. ~~Sure! I'll add that~~ if I use named exports on both we don't need a comment here
2. As far as I noticed, Vite does not handle the current implementation `export { default as useTracker }`
the following errors occur:
`'useTracker' is not exported by meteor/react-meteor-data`
`The requested module '/@id/__x00__meteor/react-meteor-data' does not provide an export named 'useTracker'`

I'll update `useTracker` and `withTracker` to use named exports. It will look clean and standardized

Thanks for the review @radekmie ❤️